__tests__/index.test.ts

```diff
@@ -627,16 +637,115 @@ describe('cast', () => {
   })
 
   test('casts binary data to array of 8-bit unsigned integers', () => {
-    expect(cast({ name: 'test', type: 'BLOB' }, '')).toEqual(new Uint8Array([]))
-    expect(cast({ name: 'test', type: 'BLOB' }, 'Å')).toEqual(new Uint8Array([197]))
-    expect(cast({ name: 'test', type: 'VARBINARY' }, 'Å')).toEqual(new Uint8Array([197]))
+    expect(cast({ name: 'test', type: 'BLOB', charset: 63 }, '')).toEqual(new Uint8Array([]))
```

Comment:
These tests all had to change to accommodate, since without specifying a charset, that's implicitly a charset of 0, which isn't even a valid thing.

If it's binary data, it'll be charset 63.